docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -48,9 +48,13 @@ here, are not included.
     - ``NULL`` in column statistic rows
   * - ``low_value``
     - The lowest value found in this column
-    - ``NULL`` in the table summary row. Available for columns of numeric types
-      (``BIGINT``, ``DECIMAL``, etc)
+    - ``NULL`` in the table summary row. Available for columns of :ref:`DATE
+      <date-time-data-types>`, :ref:`integer <integer-data-types>`,
```

Comment:
just for `DATE`, not for all types someone would see when following the link to "Date and Time" at https://trino.io/docs/current/language/types.html#date-and-time

Maybe remove the link to avoid confusion?

Reply:
Added new link and narrowed down instead